cypress/e2e/internal/billing/scenarios/change-billing-acount-current-year.cy.js

```diff
@@ -54,79 +54,18 @@ describe('Change billing account in current financial year (internal)', () => {
-      })
-      // region
-      cy.get('div:nth-child(2) > dd').should('contain.text', 'Test Region')
-      // bill run type
-      cy.get('div:nth-child(3) > dd').should('contain.text', 'Supplementary')
-      // status
-      cy.get('div:nth-child(4) > dd').should('contain.text', 'Ready')
-    })
-    cy.get('.govuk-button').contains('Send bill run').click()
-
-    // Test Region Supplementary bill run
-    // spinner page displayed whilst the bill run is 'sending'. Confirm we're on it
-    cy.get('#main-content > div:nth-child(2) > div > p.govuk-body > strong').should('contain.text', 'Sending')
-    cy.get('#main-content > div:nth-child(2) > div > p.govuk-body-l')
-      .should('contain.text', 'The bill run is being created. This may take a few minutes.')
-    cy.get('#main-content > div:nth-child(7) > div > p')
-      .should('contain.text', 'Gathering transactions for old charge scheme')
-
-    // Bill run sent
-    // confirm the bill run is sent and then click to go to it
-    cy.get('.govuk-panel__title', { timeout: 20000 }).should('contain.text', 'Bill run sent')
-    cy.get('#main-content > div > div > p:nth-child(4) > a').click()
-
-    // Test Region supplementary bill run
-    // confirm we see it is now SENT
-    cy.get('#main-content > div:nth-child(1) > div > p > strong').should('contain.text', 'Sent')
-
     // click the Bill runs menu link
     cy.get('#navbar-bill-runs').contains('Bill runs').click()
 
-    // Bill runs
-    // back on the bill runs page confirm our PRESROC bill run is present and listed as SENT
-    cy.get('@formattedCurrentDate').then((formattedCurrentDate) => {
-      cy.get('#main-content > div:nth-child(5) > div > table > tbody > tr:nth-child(1)')
-        .should('contain.text', formattedCurrentDate)
-        .and('contain.text', 'Old charge scheme')
-        .and('contain.text', 'Test Region')
-        .and('contain.text', 'Supplementary')
-        .and('contain.text', '£582.11')
-        .and('contain.text', 'Sent')
-    })
-
-    // -------------------------------------------------------------------------
-    cy.log('Confirming and sending the SROC supplementary bill run')
-
-    // select the SROC bill run
+    // we immediately select the SROC bill run. We don't expect it to be ready and to hit the spinner page but it
+    // might be super quick and already done. So we do no checks at this point
     cy.get('tr:nth-child(2) > td:nth-child(1) > a').click()
 
     // Test Region supplementary bill run
+    // we have to wait till the bill run has finished generating. The thing we wait on is the READY label. Once that
+    // is present we can confirm the bill run is a credit as expected
+    cy.get('#main-content > div:nth-child(1) > div > p > strong', { timeout: 20000 }).should('contain.text', 'Ready')
+
     // check the details before confirming the bill run
     cy.get('#main-content > div:nth-child(1) > div > p > strong').should('contain.text', 'Ready')
```

Comment:
Understand the need to insert this assertion here. But it now makes the next assertion a duplicate. So, we can just delete it.

```suggestion
    cy.get('#main-content > div:nth-child(1) > div > p > strong', { timeout: 20000 }).should('contain.text', 'Ready')
```

Reply:
Well spotted, I shall get rid of the duplicate.